src/Database/LSMTree/Internal/Lookup.hs

```diff
@@ -178,7 +174,7 @@ lookupsIO !hbio !mgr !resolveV !wb !wbblobs !rs !blooms !indexes !kopsFiles !ks
 --
 -- PRECONDITION: the vectors of bloom filters, indexes and file handles
 -- should pointwise match with the vectors of runs.
-lookupsIOWithoutWriteBuffer ::
+lookupsIO ::
```

Comment:
I removed the `Note` in an earlier PR, but forgot to remove the reference in the comment here

Reply:
I'll remove the reference.